src/test/testing-library/AuthButtons.test.tsx

```diff
@@ -1,9 +1,13 @@
 import { render } from '@testing-library/react';
 
+import { createTranslation, TranslationFunctionType } from '@/test/translate';
+
 import AuthButtons from '../../features/landing/components/Header/AuthButtons/AuthButtons';
 
-const buttonLogInTestId: string = 'Log in';
-const buttonSignUpTestId: string = 'Try it out';
+const t: TranslationFunctionType = createTranslation('pages/i18n');
```

Comment:
bad name for type

Reply:
by default we already know in which place our localization file, need to set default option in function and move it to env